source/frontend/src/components/RegionManagement/DiscoverableRegions/DiscoverableAccountsTable.js

```diff
@@ -127,49 +255,70 @@ const DiscoverableAccountsTable = ({ selectedAccounts, onSelect }) => {
       >
         Remove the AWS account <strong>{selectedAccounts.map(i => R.defaultTo(i.accountId, i.accountName)).join(", ")}</strong>?
       </Modal>
-      <Table
-        {...collectionProps}
-        header={
-          <Header
-            actions={
-              isUsingOrganizations()
-                  ? void 0
-                  : <SpaceBetween direction='horizontal' size='xs'>
-                      <Button
-                          disabled={R.isEmpty(selectedAccounts)}
-                          onClick={() => setShowDeleteConfirm(true)}>
-                          Remove
-                      </Button>
-                      <Button
-                          loadingText='Removing'
-                          variant='primary'
-                          onClick={(e) => {
-                              e.preventDefault();
-                              history.push(IMPORT);
-                          }}>
-                          Import
-                      </Button>
-                  </SpaceBetween>
-            }
-            variant='h2'
-            description='AWS accounts that contain Regions imported into Workload Discovery on AWS'>
-            Accounts
-          </Header>
-        }
-        loading={isLoading}
-        trackBy={'id'}
-        resizableColumns
-        columnDefinitions={columns}
-        items={items}
-        selectedItems={selectedAccounts}
-        selectionType='multi'
-        onSelectionChange={(evt) => onSelectionChange(evt.detail.selectedItems)}
-        loadingText='Loading accounts'
-        filter={
-          <TextFilter {...filterProps} filteringPlaceholder='Find an Account...' />
-        }
-        pagination={<Pagination {...paginationProps} />}
-      />
+      <SpaceBetween size="s">
+          {R.isEmpty(noIamDeployedAccounts)
+              ? void 0
```

Comment:
Why `void 0`?
Could you do 
```js
{!R.isEmpty(noIamDeployedAccounts) && <AccountAlert globalTemplate={globalTemplate} accounts={accounts}/>
```

Reply:
This is more a question for my own understanding

Reply:
In JavaScript `undefined` is not a reserved keyword, so it's actually possible to create a variable called `undefined`. The `void` keyword always ensures the the value return is really `undefined`. I think most modern browser disallow globally changing the value of `undefined` but you can still do it locally.